site/documentation/content/user-guide/device-registry.md

```diff
@@ -403,7 +398,7 @@ ETag: d383ba4d-1853-4d03-b322-b7ff5af05ae2
 
 **Example**
 
-The following command retrieves credentials for device `4720`:
+The following command retrieves credentials metadata for device `4720`:
```

Comment:
I guess the example payload should at least contain two secrets with e.g. adjoining validity periods so that you can see, what the *id* would be useful for (e.g. for changing one of the two passwords) ...

Reply:
Good idea ! I added another secrets with `not-after` and `not-before` overlapping with only one day.